usl_pipeline/cloud_functions/main.py

```diff
@@ -272,9 +274,17 @@ def build_feature_matrix(cloud_event: functions_framework.CloudEvent) -> None:
     GCS. It produces a feature matrix for the geo data and writes that feature matrix to
     another GCS bucket for eventual use in model training and prediction.
     """
+    _build_feature_matrix(
+        cloud_event.data["bucket"],
+        cloud_event.data["name"],
+        cloud_storage.FEATURE_CHUNKS_BUCKET,
+    )
+
+
+def _build_feature_matrix(bucket_name, chunk_name, output_bucket) -> None:
```

Comment:
let's add type annotations for these function arguments

Reply:
oh, good catch. It's weird that I still keep forgetting about type hints. :(